lib/task.ex

```diff
@@ -9,13 +9,34 @@ defmodule Detektor.Task do
     {:ok, state}
   end
 
+  def fetchPlaylist(url, parent, _) do
+    cmd = "youtube-dl"
+    args = [ url, "--flat-playlist", "-J"]
+
+    Logger.debug"> youtube-dl #{Enum.join(args, " ")}"
+    case System.cmd(cmd, args, [parallelism: true, stderr_to_stdout: true]) do
+      {output, 0} ->
+        {:ok, youtubeHash}  = output |> String.split("\n") |> hd |> Poison.decode
+
+        constructYoutubeURL = fn(id) -> "https://www.youtube.com/watch?v=#{id}" end
```

Comment:
This is constraining us to only youtube. I wonder if we can get the full url instead of only the track ID.